bin/cordova_plist_to_config_xml

```diff
@@ -106,7 +106,7 @@ def UpdateProjectFile(path):
     if 'Cordova.plist' in line:
       line = line.replace('Cordova.plist', 'config.xml')
       line = line.replace('lastKnownFileType = text.plist.xml', 'lastKnownFileType = text.xml')
-    print(line, end='')
+    print(line, end=' ')
```

Comment:
Why add a space here?

Reply:
To put a space between each line.

Reply:
I don't really understand (did not work with Python for about 10 years), hope someone else can review it. We do appreciate your efforts to get this whole Python compatibility issue fixed.

Reply:
Which do we want?
```
>>> for char in "abcdefg":
...     print(char, end="")  # --> abcdefg
    -- or --
>>> for char in "abcdefg":
...     print(char, end="")  # --> a b c d e f g
```